src/components/map/Map.tsx

```diff
@@ -36,18 +38,29 @@ type Props = {
 const MARKER_ICON_SIZE = normalize(40);
 const CIRCLE_SIZES = [60, 50, 40, 30];
 
-const MapIcon = ({
-  iconColor,
+export const MapIcon = ({
+  iconColor = colors.lighterPrimary,
   iconName = 'location',
-  iconSize = MARKER_ICON_SIZE
+  iconSize = MARKER_ICON_SIZE,
+  iconStrokeColor = colors.darkerPrimary,
```

Comment:
When I click on Cluster, it zooms out and goes to the center of the world map.

Reply:
thanks, fixed. we will need to take care when merging back to MQGB somewhen:

![image](https://github.com/user-attachments/assets/9a6f4b0c-63e1-46ed-998d-5434c919875b)

this is used in MQGB to exclude the own location marker from clustering.